codespell_lib/data/dictionary.txt

```diff
@@ -17459,6 +17493,7 @@ inititalisation->initialisation
 inititalisations->initialisations
 inititalization->initialization
 inititalizations->initializations
+inititalize->initialize
```

Comment:
Can you also separately add the ise versions.

Reply:
01b1a64